### PR TITLE
Update kinematic-chain-config.md

### PR DESCRIPTION
**Pull request by cheukt**
# Description

Drive by, not sure if I am placing this in the right place/wording it right. The kinematics information is exposed in the SDKs for arms and also the frame system itself is exposed through FrameSystemConfig().

### CODE REVIEW

File: docs/internals/kinematic-chain-config.md [hunk 1]
```diff
@@ -311,7 +311,7 @@ This is an example <file>.json</file> configuration as used by Viam's [Universal
 
 {{% alert title="Info" color="info" %}}
 
-These reference frames are ingested by the Frame System, but not exposed through gRPC calls.
-You cannot access them directly with SDK code.
+These reference frames are ingested by the Frame System, but not exposed through gRPC calls except for arms.
+For arms, you can use the GetKinematics() call to access kinematics information of the arm.
```
Comment (sguequierre) [inline docs/internals/kinematic-chain-config.md:315]:
```suggestion
If your resource is an [arm component](/components/arm/), you can use the [Kinematics()]((/components/arm/#kinematics)) method to access its kinematics information.
```

File: docs/internals/kinematic-chain-config.md [hunk 2]
```diff
@@ -311,7 +311,7 @@ This is an example <file>.json</file> configuration as used by Viam's [Universal
 
 {{% alert title="Info" color="info" %}}
 
-These reference frames are ingested by the Frame System, but not exposed through gRPC calls.
-You cannot access them directly with SDK code.
+These reference frames are ingested by the Frame System, but not exposed through gRPC calls except for arms.
```
Comment (sguequierre) [inline docs/internals/kinematic-chain-config.md:314]:
```suggestion
These reference frames are ingested by the Frame System.
They are not exposed in the [client SDKs](/program/), with one exception.
```
[opt] if this is accurate

**Review by sguequierre — APPROVED** [review]
thank you for updating this!! I added a couple of suggestions, other than that lgtm once jessamy gets a chance to look over as well

**Comment by JessamyT** [comment]
@sguequierre awesome, I was just going to add you because I assume this relates to https://viam.atlassian.net/browse/DOCS-854?

**Comment by cheukt** [comment]
thanks for the suggestions @sguequierre !

**Review by JessamyT — APPROVED** [review]
LGTM if it looks good to y'all

File: docs/internals/kinematic-chain-config.md [hunk 3]
```diff
@@ -311,7 +311,8 @@ This is an example <file>.json</file> configuration as used by Viam's [Universal
 
 {{% alert title="Info" color="info" %}}
 
-These reference frames are ingested by the Frame System, but not exposed through gRPC calls.
-You cannot access them directly with SDK code.
+These reference frames are ingested by the Frame System.
+They are not exposed in the [client SDKs](/program/), with one exception.
+If your resource is an [arm component](/components/arm/), you can use the [Kinematics()]((/components/arm/#kinematics)) method to access its kinematics information.
```
Comment (sguequierre) [inline docs/internals/kinematic-chain-config.md:316]:
```suggestion
If your resource is an [arm component](/components/arm/), you can use the [Kinematics()](/components/arm/#kinematics) method to access its kinematics information.
```

**Pull request merged**